EmployeesSalaries/EmployeesSalaries/Models/Employee/Dev.cs

```diff
@@ -7,9 +7,22 @@ public class Dev : IEmployee, IReportsTo
         public string FirstName { get; set; } = "Test";
         public string LastName { get; set; } = "Test";
         public string Role { get; set; } = "Dev"; 
```

Comment:
there's no need for Role, you already know what employee type, since you already in the Dev class